### ISSUE TITLE
a11y issue: toast-manager elements injected - The element with "region" role is not labeled with an 'aria-label' or 'aria-labelledby'

### ISSUE THREAD

**Issue by f1729** [chakra-ui/chakra-ui#7831]
### Description

I'm getting this error after a a11y scan for my site, I'm not sure if this is a real issue, but in any case what would be a workaround to avoid this "issue"?

- Even without using toast these elements got injected, so you can find it in any Chakra project.

<img width="812" alt="Screenshot 2023-07-06 at 6 24 00 PM" src="https://github.com/chakra-ui/chakra-ui/assets/23299193/78708d4b-70ac-4fdb-ad67-955ef4a53e08">
<img width="1709" alt="Screenshot 2023-07-06 at 6 24 46 PM" src="https://github.com/chakra-ui/chakra-ui/assets/23299193/086e10cc-b92d-4ff9-93ab-f179f0ed98ea">

### Link to Reproduction

https://codesandbox.io/s/chakra-ui-next-js-typescript-kxvyr

### Steps to reproduce

1. Go to any Chakra project
2. Open developer tools
3. Look for 'chakra-toast-manager'
4. See all the element 

### Chakra UI Version

2.7.0

### Operating System

- [X] macOS
- [X] Windows
- [X] Linux

**Comment by segunadebayo** [comment]
Thanks for reporting this issue.

I just pushed a fix for this. We'll release an update shortly.

If the issue persists after upgrading, I'll re-open it.

**Comment by tpiere** [comment]
@segunadebayo  we have the latest version (2.8.0) and now we get a different but related error on our a11y scan:

![Screenshot 2023-09-06 at 3 40 37 PM](https://github.com/chakra-ui/chakra-ui/assets/1138652/1b49bbfa-52d0-4079-8fa0-d32b1c7af5c8)

All of the toast manager regions have the same `aria-label` ("Notifications") which it's flagging since they need to be unique

**Comment by bluecoder5015** [comment]
> @segunadebayo we have the latest version (2.8.0) and now we get a different but related error on our a11y scan:
> 
> ![Screenshot 2023-09-06 at 3 40 37 PM](https://user-images.githubusercontent.com/1138652/266148409-1b49bbfa-52d0-4079-8fa0-d32b1c7af5c8.png)
> 
> All of the toast manager regions have the same `aria-label` ("Notifications") which it's flagging since they need to be unique

Any update on this a11y accessibility issue ?

**Comment by Isaac-Leonard** [comment]
I'm a screen reader user and just started a new project with chakra ui.
I've used this library before with no issues like this but my most recent one seems to have this.
I'm using safari on macos and version 2.8.2
Currently I'm trying to work out how to best remove the elements manually on page load but this is definitely an accessibility issue as I get 8 or so extra elements on the page that just say "Notifications-top empty region", "Notifications-top-left empty regions", etc

**Comment by Isaac-Leonard** [comment]
It also screws with focus for me taking focus off of my app and putting it on the toast elements
I found I can remove them by adding:
```
  React.useEffect(() => {
    document.querySelectorAll(".chakra-portal")?.forEach((x) => x.remove());
  }, []);
```
to my App component

**Comment by edtech1985** [comment]
I'm having the same issue as f1729.
Was it fixed? Could not find anything about it in documentation.
